### PR TITLE
JAMES-3440 JMAP tasks integration tests on top of the Distributed Server

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java [hunk 1]
```diff
@@ -156,6 +156,64 @@ void fullReindexingShouldCompleteWhenNoMail() {
...
+                MailboxPath.inbox(Username.of(USERNAME)),
+                new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes()),
+                new Date(),
+                false,
+                new Flags());
+
+        String taskId = with()
+            .post("/mailboxes?task=recomputeFastViewProjectionItems")
+            .jsonPath()
+            .get("taskId");
```
Comment (LanKhuat) [inline server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java:174]: indent?

File: server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java [hunk 2]
```diff
@@ -156,6 +156,64 @@ void fullReindexingShouldCompleteWhenNoMail() {
...
+    void recomputeFastViewProjectionItemsShouldCompleteWhenNoMail(GuiceJamesServer server) throws Exception {
+        server.getProbe(DataProbeImpl.class).addUser(USERNAME, "secret");
+        mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, USERNAME, MailboxConstants.INBOX);
+        mailboxProbe.appendMessage(
+                USERNAME,
+                MailboxPath.inbox(Username.of(USERNAME)),
+                new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes()),
+                new Date(),
+                false,
+                new Flags());
```
Comment (LanKhuat) [inline server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java:169]: There is a mail though..

File: server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java [hunk 3]
```diff
@@ -156,6 +156,64 @@ void fullReindexingShouldCompleteWhenNoMail() {
...
+    void populateEmailQueryViewShouldCompleteWhenNoMail(GuiceJamesServer server) throws Exception {
+        server.getProbe(DataProbeImpl.class).addUser(USERNAME, "secret");
+        mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, USERNAME, MailboxConstants.INBOX);
+        mailboxProbe.appendMessage(
+                USERNAME,
+                MailboxPath.inbox(Username.of(USERNAME)),
+                new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes()),
+                new Date(),
+                false,
+                new Flags());
```
Comment (LanKhuat) [inline server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java:198]: idem

File: server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java (hunk 1 above)
Comment (chibenwa) [reply server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java:174]: Nope we indent given/when/then not with calls.

**Review by LanKhuat — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**